Lombiq.Walkthroughs/ResourceManagementOptionsConfiguration.cs

```diff
@@ -13,11 +13,11 @@ static ResourceManagementOptionsConfiguration()
     {
         _manifest.DefineResource("$" + nameof(FeatureIds.Area), FeatureIds.Area);
 
+        // We can't set shepherd.min.css here, since the file comes from a node module and it's originally a css file,
+        // so we are not processing it. It's also somewhat minified by default.
```

Comment:
You can reference files from _node_modules_ and include them in the wwwroot. Many of our modules do this, e.g. `Lombiq.ChartJs`.

Reply:
I'm already doing exactly that: https://github.com/Lombiq/Orchard-Walkthroughs/blob/480ebb7d3e780ee45887efbcb9633745b2efa8d9/Lombiq.Walkthroughs/package.json#L14-L25

Reply:
This explanation is about how I can't use the usual

```
                "~/" + FeatureIds.Area + "/shepherd.js/css/shepherd.min.css",
                "~/" + FeatureIds.Area + "/shepherd.js/css/shepherd.css");
```
pattern, since in the node_modules, only `shepherd.css` exists but no `scss`, so it won't go trough the pipeline (it only gets copied) (the pipeline looks for `scss` files), thus there will be no `shepherd.min.css`.

Reply:
I see. The comment is misleading for me since it make it appear that there is actually an shepherd.min.css. I added a suggestion.

Reply:
Cool, thanks!